### DISCUSSION TITLE
RestoreEntityHandler::process(100070686833289951): lose space

### DISCUSSION THREAD

**Discussion by thinbug** [kbengine/kbengine#1546] [General]
运行一段时间后出现了大量的日志,里面全部都是下面的log,直到把服务器硬盘写满,

请问有没有朋友也出现过这个情况?

不知道是什么原因引起的
 
 ERROR root [0x7f06d52450c0] [2022-02-08 18:41:49,642] - RestoreEntityHandler::process(100070686833289951): lose space(1711940).
 ERROR root [0x7f06d52450c0] [2022-02-08 18:41:49,642] - RestoreEntityHandler::process(100070686833289951): lose space(1711965).
 ERROR root [0x7f06d52450c0] [2022-02-08 18:41:49,642] - RestoreEntityHandler::process(100070686833289951): lose space(1707043).
 ERROR root [0x7f06d52450c0] [2022-02-08 18:41:49,642] - RestoreEntityHandler::process(100070686833289951): lose space(1714484).
 ERROR root [0x7f06d52450c0] [2022-02-08 18:41:49,642] - RestoreEntityHandler::process(100070686833289951): lose space(1677476).
 ERROR root [0x7f06d52450c0] [2022-02-08 18:41:49,642] - RestoreEntityHandler::process(100070686833289951): lose space(1662666).
 ERROR root [0x7f06d52450c0] [2022-02-08 18:41:49,642] - RestoreEntityHandler::process(100070686833289951): lose space(1673384).
 ERROR root [0x7f06d52450c0] [2022-02-08 18:41:49,642] - RestoreEntityHandler::process(100070686833289951): lose space(1718676).
 ERROR root [0x7f06d52450c0] [2022-02-08 18:41:49,642] - RestoreEntityHandler::process(100070686833289951): lose space(1717857).

**Comment by huhubis** [comment]
cellapp挂了？
